.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -21,12 +17,29 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+      - name: Setup Node.js (.nvmrc)
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+      - name: Install npm dependencies
+        run: npm ci
+      - name: Get plugin version
+        id: get-version
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo "version=$(node ./bin/plugin/cli.js get-plugin-version --slug=${{ inputs.slug }})" >> $GITHUB_OUTPUT
       - name: Set matrix
         id: set-matrix
         run: |
           if ${{ github.event_name == 'workflow_dispatch' }}; then
-            # Set the manual input values in JSON format for use in the matrix.
-            echo "matrix={\"include\":[{\"slug\":\"${{ inputs.slug }}\",\"version\":\"${{ inputs.version }}\"}]}" >> $GITHUB_OUTPUT
+            if ${{ steps.get-version.outputs.version == '' }}; then
+              echo "The ${{ inputs.slug }} module slug is missing in the file plugins.json."
```

Comment:
Should we validate that the string is a valid version number?

That may be needed if e.g. something was wrong with the CLI script and it resulted in incorrect output. I would assume the above step would already fail and terminate with an error in that case? But if not, or maybe even just to be safe it may be good to add a regex to validate here.

For example, this regex would work: `^(\*|\d+(\.\d+){0,2}(-[a-zA-Z0-9\.]+)?)$` (covers stable versions as well as beta and RC naming conventions)

Reply:
Thanks @felixarntz for the feedback. In https://github.com/WordPress/performance/pull/710/commits/4995c95b38eeacb3104e2a15e46afa72f8bdf4d5 i have address it.